dandi/models.py

```diff
@@ -179,6 +181,15 @@ class SpeciesType(TypeModel):
 class Disease(TypeModel):
     """Biolink, SNOMED, or other identifier for disorder studied"""
 
+    dxdate: Optional[List[date]] = Field(
+        None,
+        title="Dates of diagnosis",
+        description="Dates of diagnosis",
+        readOnly=True,
+        nskey="dandi",
+        rangeIncludes="schema:Date",
+    )
+
```

Comment:
hm, shouldn't this (and possibly other changes) have affected the produced json schema?  according to the workflow no changes being pushed to `dandi/schema` repository by https://github.com/dandi/dandi-cli/runs/1476406805?check_suite_focus=true

Reply:
it should have.

Reply:
filed #289 289